create-aleo-app/template-extension/rollup.config.js

```diff
@@ -0,0 +1,20 @@
+import { nodeResolve } from "@rollup/plugin-node-resolve";
+import { importMetaAssets } from "@web/rollup-plugin-import-meta-assets";
+
+export default {
+    input: {
+        background: "./src/background.js",
+        worker: "./src/worker.js",
```

Comment:
Can we document what these are/what their function is?

Reply:
I guess we could, but it's super standard Chrome extension stuff, which is common knowledge for anybody creating a Chrome extension.

https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/background

The sample is really simple, anybody who is familiar with Chrome extensions should understand it right away.